coreos-bootstrap.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: k8s-cluster
+  gather_facts: False
+  roles:
+    - coreos-bootstrap
```

Comment:
Actually this is not possible because the condition `when: ansible_os_family != "CoreOS"` uses a fact.
The fact are disabled because python is not installed.